packages/next-codemod/transforms/add-missing-react-import.ts

```diff
@@ -0,0 +1,77 @@
+function addReactImport(j, root) {
+  // We create an import specifier, this is the value of an import, eg:
+  // import {withRouter} from 'next/router
+  // The specifier would be `withRouter`
+  const ReactDefaultSpecifier = j.importDefaultSpecifier(j.identifier('React'))
```

Comment:
I don't believe this is correct, we should be generating:

```tsx
import * as React from 'react'
```

As it results in smaller build output (less webpack wrapping) and higher performance (no object property access in render).